setup.cfg

```diff
@@ -2,6 +2,11 @@
 name = aiomysql
 url = https://github.com/aio-libs/aiomysql
 download_url = https://pypi.python.org/pypi/aiomysql
+project_urls =
```

Comment:
Mind putting this in a separate PR? + maybe add a changelog link too?

Reply:
created #738 to add the changelog to docs and then also reference it here

Reply:
moved this change to #739